.github/workflows/interface-unit-tests.yml

```diff
@@ -166,7 +166,8 @@ jobs:
       install_pennylane_lightning_master: true
       pytest_coverage_flags: ${{ inputs.pytest_coverage_flags }}
       pytest_markers: torch and not qcut and not finite-diff and not param-shift
-
+      env: # TODO: remove this after Catalyst v0.4.0 release
+        MIN_CATALYST_VERSION: "0.3.2"
```

Comment:
I feel weird adding the handling of this environment variable to source code if it's only for this test suite. Is that the case here? Also, is this intended to only be in `torch-tests`?

Reply:
This env variable is added to `external-libraries-tests` [0f32cb9](https://github.com/PennyLaneAI/pennylane/pull/4979/commits/0f32cb93db6b1f112c7db1bbc561a20b61370d6f) to temporary alter the minimum version of Catalyst for tests in `test_compiler.py`. This will be removed after Catalyst v0.4.0 release next week.

Reply:
does this need to be in the 0.34 release then? we've already made the RC branch, so if so, please target it (`v0.34.0-rc0`) and add this PR to the 0.34 milestone to make sure we don't miss it.

Either way, is the plan after the Catalyst 0.4.0 release to hard-code the minimum to be 0.4.0?

Reply:
Thanks @timmysilv! I'll do rebase the PR. The minimum version of Catalyst is set to 0.4.0 in `compiler.py`, but I added this env variable to enable us to check the PL-Catalyst tests in `test_compiler.py` before releasing Catalyst v0.4.0. We can get rid of this env variable next week after the release.